tests/election.py

```diff
@@ -7,7 +7,7 @@
 import math
 import infra.ccf
 import infra.proc
-import infra.jsonrpc
+import infra.client
```

Comment:
If the only change is renaming an import, then we're not actually using the import in this file - so many of these can simply be removed?

Reply:
You're absolutely right!